pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -222,15 +222,27 @@ void processArguments() throws Exception {
                 description = "Path to the main Python file for the function (if the function is written in Python)",
                 listConverter = StringConverter.class)
         protected String pyFile;
-        @Parameter(names = "--inputs", description = "The function's input topic or topics (multiple topics can be specified as a comma-separated list)")
+        @Parameter(
+            names = "--inputs",
+            description =
+                "The function's input topic or topics (multiple topics can be specified as a comma-separated list)."
+                + " A default SerDe is used for serializing/deserializing messages. If you want to use your own SerDe,"
+                + " specify the topic and the SerDe class in [--customSerDeInputs]. The total list of input topics is"
+                + " a union of [--inputs] and ([--topicsPattern]/[--customSerDeInputs])."
+                + " A topic should only be specified in either [--inputs] or [--customSerDeInputs].")
```

Comment:
if you are specifying a topic, it can only be in one place, either --inputs or --customSerDeInputs.

If you are specifying a topic pattern, it has to be in --topicsPattern; if you are using a topic pattern and want to use a customized serde, the topic pattern needs to be specified in --customerSerDeInputs.

any suggestions on improving this?

Reply:
So --customSerDeInputs can take a pattern as well? do you need to do
```--customSerDeInputs <topicPattern> --topicsPattern <topicPattern>```
or is
```--customSerDeInputs <topicPattern>``` sufficient?

Reply:
I think you need `--topicsPattern` to specify the pattern. because from --customSerDeInputs, it is hard to tell whether it is a pattern or topic.

Reply:
That's very confusing then, because ```--topicsPattern``` can duplicate ```--customSerDeInputs``` but ```--topics``` can not. Couldn't we just add dedupe logic to --customSerDeInputs?